src/test/regress/expected/multi_mx_insert_select_repartition.out

```diff
@@ -90,6 +104,16 @@ SELECT * FROM target_table ORDER BY a;
  5 | 400
 (6 rows)
 
+SET citus.log_local_commands to on;
+BEGIN;
```

Comment:
I guess the point there is `shard_count` is set to 4 before creating that table, and that table has only one shard on the coordinator. This is way it is executed locally there but remotely in mx node

Reply:
I'm planning to increase number of shards a bit and make this table colocate with = 'none', does it sound good ?

Reply:
fixed, I aware that it was a router query previously and so we were trying to perform local execution because of that

Reply:
I don't think it is fixed? Now, we're starting with a multi-shard query, which prevents local execution. We should start with a local router query, right? Our goal was to show that a local router query followed by INSERT ... SELECT via coordinator / via repartition cause query failures.


 We should update the test with  `insert into target_table SELECT a*2 FROM source_table;`  as well.

Reply:
I changed it now again but it seems that that transaction block still does not fail from master. It's probably because we have only 1 shard in coordinator and citus prefers to use local execution. I will change the shard count before creating that table